cypress/TestCases/FireboltCertification/UserInterest.feature

```diff
@@ -0,0 +1,113 @@
+Feature: UserInterest
+
+    @initialization
+    Scenario: Launch FCA for 'Userinterest'
+        Given the environment has been set up for 'Userinterest' tests
+        And 3rd party 'certification' app is launched
+
```

Comment:
And '3rd party app' invokes the 'Firebolt' API to 'get userInterest with <GetParam>' 
to be changed to 
And '3rd party app' invokes the 'Firebolt' API to 'express userInterest with <GetParam>'

Reply:
i would say use **_notify_** instead of express... would make more sense

and also it is not GetParam but something like **_details_**

Reply:
corrected